regression/pages/ecommerce/cybersource_page.py

```diff
@@ -0,0 +1,111 @@
+"""
+Cybersource page
+"""
+from bok_choy.page_object import PageObject
+
+from regression.pages.common.utils import (
+    fill_input_fields,
+    select_values_from_drop_downs,
+    click_checkbox
+)
+from regression.pages.ecommerce.back_to_basket_page import BackToBasketPage
+from regression.pages.ecommerce.cancel_checkout_page import CancelCheckoutPage
+
+
+class CyberSourcePage(PageObject):
+    """
+    Cybersource payment page
+    """
+
+    url = 'https://testsecureacceptance.cybersource.com/checkout'
+
+    def is_browser_on_page(self):
+        """
+        Is browser on the page?
+        Returns:
+            True if payment button is present:
+        """
+        return self.q(css='.right.complete.pay_button').present
+
+    def set_billing_info(self, bill_info):
+        """
+        Fill billing form using the values passed from test
+        Args:
+             bill_info:
+        """
+        fill_css_selectors = [
+            '#bill_to_forename',
+            '#bill_to_surname',
+            '#bill_to_address_line1',
+            '#bill_to_address_line2',
+            '#bill_to_address_city',
+            '#bill_to_address_postal_code',
+            '#bill_to_email'
+        ]
+        fill_values = [
+            bill_info['first_name'],
+            bill_info['last_name'],
+            bill_info['address01'],
+            bill_info['address02'],
+            bill_info['city'],
+            bill_info['postal_code'],
+            bill_info['email']
+        ]
+        select_names = [
+            "bill_to_address_country",
+            "bill_to_address_state_us_ca"
+        ]
+        select_values = [bill_info['country'], bill_info['state']]
+        fill_input_fields(self, fill_css_selectors, fill_values)
```

Comment:
yeah definitely use a dict rather than two independent lists that need to line up in the same order.